victoriametrics-operator/Dockerfile

```diff
@@ -3,8 +3,18 @@
 # Stage1: build from source
 FROM quay.io/cybozu/golang:1.17-focal AS build
 
-ARG VICTORIAMETRICS_OPERATOR_SRCREPO=VictoriaMetrics/operator
-ARG VICTORIAMETRICS_OPERATOR_VERSION=0.18.2
+#ARG VICTORIAMETRICS_OPERATOR_SRCREPO=VictoriaMetrics/operator
+#ARG VICTORIAMETRICS_OPERATOR_VERSION=0.18.2
+ARG VICTORIAMETRICS_OPERATOR_SRCREPO=cybozu-go/VictoriaMetrics-operator
+ARG VICTORIAMETRICS_OPERATOR_VERSION=0.19.1-neco.1
```

Comment:
Would you delete tags `0.19.1-neco.2` and `0.19.1-neco.3` if unnecessary?
Their names indicate as if they were newer and more appropriate.  But based on my investigation they are patching to inappropriate commits.

Reply:
OK. Deleted.